docs/finalproject/external-projects.md

```diff
@@ -0,0 +1,127 @@
+
+3. Be well-defined
+
+   - This means that from day one our students understand what they are building
+   - For example, a well defined project should be able to answer using [User Stories](https://www.mountaingoatsoftware.com/agile/user-stories) **all** of the functionality of the project
+   - We **can** work with Charities to define projects if they contact us early enough
+
+### Technical Functionality
+
+_Don't know what anything here means? That's fine! The above section covers this in less technical details_
+
+We use the following technologies at CodeYourFuture
+
+- JavaScript
+- React
+- Node with Express
+- MongoDB or PostgreSQL
+
+An external Final Project should use all of these technologies into a single Full Stack package. This is important as we want our students to use all of the skills that they have learnt in our course.
+
+We advise against using other tools as they will fall outside our students areas of competency due to the time constraints that we have.
+
+### Limitations
+
+As Junior Developers, our students do have some limits on what they can produce
+
+- Visual Design Complexity
+  - An overly complex design will take a lot longer for our students to develop. Try to keep your designs simple and functional.
+- Instant Answers
+  - You shouldn't expect our students to be able to give instant answers to any technical questions you might have. They will get there with time however!
+
+## Next Steps
+
+### 1) Details Form
+
+A good next step to getting started with defining your project is to complete this form.
+
+[CodeYourFuture - Projects Specification Form](https://docs.google.com/forms/u/1/d/1u2zImwEjD5PDWJuMyDeCuQ3AP5xhWr683pGy9HlmC14/edit)
+
+It is important that you answer all of these questions in **as much** details as possible. Even irrelevant details will help us understand what you are trying to achieve.
+
+### 2) Visual Form
+
+On the new step we're asking you to be more visual in answering the questions. We'd advise you to use images or diagrams to answer the questions in these slides.
+
+[CodeYourFuture - Visual Specification Form](https://docs.google.com/presentation/d/1-rj0NAncIhIZuay6TXEyUeuXFFUnE7vmN12223rHZOI/edit#slide=id.g3579738be0_0_0)
+
+### 3) Design Mock-Ups
+
+Finally, you should work with us to create some simple designs of your product. These will not be the final designs of what you be building - they're just a tool to help us work out what you want built.
+
+Very quick designs of the tool/website you are trying to build will give us a great insight into what exactly you want built.
+
+You needn't use any specialised tools to design this. Paint or Pen & Paper will do for this step.
+
+Try to think about how each individual page will look. Will the user need to create an account? Think of the form they will need to fill in. Does the user view content on a page? What content should be included?
+
+Get in contact with your local City Coordinator to arrange this.
```

Comment:
Do we have an contact page or something we could link to?

Reply:
By this point they should have contact with their City Coordinator - I imagine this page will be sent to the NGOs directly rather than them finding it by themselves